content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -106,11 +129,24 @@ Product-specific dynamic text values are as follows:
       :alt: PDF quote being built using common dynamic placeholders.
 
 Once the PDF file(s) are complete, save them to the computer's hard drive, and proceed to upload
-them to Odoo via :menuselection:`Sales app --> Configuration --> Settings --> PDF Quote builder`.
+them to Odoo via :menuselection:`Sales app --> Configuration --> Headers/Footers`.
 
-Upload the created PDF in the :guilabel:`Header pages` or :guilabel:`Footer pages` field.
+.. example::
+   When uploading a header PDF containing the form field :guilabel:`invoice_partner_country`,
+   configuring the :guilabel:`path` of the :guilabel:`Form Field Name` to
+   :guilabel:`partner_invoice_id.country_id.name` fills the form with the invoice partner country's
+   name when the PDF is built.
 
-Once the upload(s) are complete, click :guilabel:`Save`.
+.. example::
+   When uploading a product PDF containing the form field :guilabel:`invoice_partner_country`,
+   configuring the :guilabel:`path` of the :guilabel:`Form Field Name` to
+   :guilabel:`order_id.partner_invoice_id.country_id.name` fills the form with the invoice partner
+   country's name when the PDF is built.
```

Comment:
I believe these two examples are duplicates.

Reply:
The difference between these two is that one is a header document, whereas the other is a product document. I voluntarily used the same final information to point the different path, as the path is different for product or header/footer document.
I added some formatting to improve the readability/difference between the two.